core/node/state_keeper/src/updates/mod.rs

```diff
@@ -39,6 +39,7 @@ pub struct UpdatesManager {
     pub l2_block: L2BlockUpdates,
     pub storage_writes_deduplicator: StorageWritesDeduplicator,
     pubdata_params: PubdataParams,
+    pub next_l2_block_param: L2BlockParams,
```

Comment:
- Please make this field an `Option` to align with semantics; we don't always have the next block params, only sometimes. These params would be set by `set_next_l2_block_parameters()` (asserting that they are `None`, i.e. we never redefine next block params from scratch) and `take()`n by `push_l2_block()`.
- AFAIU, there's no need to make this field public. Some state keeper logic reads a timestamp, but this can be done via `next_l2_block_timestamp()`. Making the field private ensures that its intended lifecycle invariants above are upheld.